### DISCUSSION TITLE
How to rollback to a specific version

### DISCUSSION THREAD

**Discussion by devstress** [asimeonov/DbUp.Downgrade#16] [Q&A] [answered]
My folder structure is like this:
* Scripts                            
    * Script0001 - Create tables.sql          
    * Script0002 - Default feed.sql
    * Script0003 - Settings.sql
    * Script0004 - Transform data.sql
* DowngradeScripts                    
    * Script0002 - Default feed.sql
    * Script0003 - Settings.sql
    * Script0004 - Transform data.sql
    
I want to roll back to a specific version such as the state after executing Script0002 - Default feed.sql. How can I do that?

**Comment by asimeonov** [comment, marked as answer]
Hi @devstress,

I'll assume that you are using Embedded resource scripts. So they that library is working is by comparing currently executing version against the version schema stored in the DbUp journal table. In order to revert the chenges from `Scripts\Script0003 - Settings.sql` and `Scripts\Script0004 - Transform data.sql` you need to remove them wither by removing the file or by removing them as an Embedded resource.
At the moment there is no manual way to remove them like in EF. In futre I can write a tool to do that.

**Reply by devstress** [reply]
Thanks @asimeonov. Looking forward to the next version to support the manual rollback to a specific version.
